pkg/grpcfactory/grpc_factory_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package grpcfactory_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestCosiDev(t *testing.T) {
```

Comment:
Maybe rename this function to `TestGRPCFactorySuite`?

Reply:
done